packages/e2e-tests/examples.json

```diff
@@ -5,54 +5,54 @@
     "requiresManualUpdate": true
   },
   "authenticated_comments.html": {
-    "recording": "b50dd5b7-ab70-4987-8987-cc3240dd2cf3",
-    "buildId": "macOS-chromium-20240110-e5625ebd3edc-50d01be708a2"
+    "recording": "94775d4e-33e7-4c02-8622-3b212503ebab",
+    "buildId": "linux-chromium-20240123-609ec6caa105-50d01be708a2"
```

Comment:
I think we should revert the changes tot his file. They aren't really a part of this change set, are they?